src/pkgdev/scripts/pkgdev_commit.py

```diff
@@ -80,6 +84,29 @@ def __call__(self, parser, namespace, value, option_string=None):
         namespace.footer.add((name.capitalize(), val))
 
 
+class BugzillaAwareBugTag(BugTag):
+    """Register bug-related tag and resolution to inject into the commit message footer."""
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        try:
+            bug, res = values.rsplit(":", 1)
+            if res.startswith("//"):
+                raise ValueError("URL-like value without resolution")
+            res = res.lower() if res else const.BZ_FIXED
+            if not (bug and res in const.BZ_RESOLUTIONS):
+                raise argparse.ArgumentError(self, f"invalid commit tag: {values}")
+        except ValueError:
+            super().__call__(parser, namespace, values, option_string)
+            return
+
+        url = self.parse_url(bug)
+        is_bgo = url.split("//", 1)[1].startswith("bugs.gentoo.org")
```

Comment:
```suggestion
        is_bgo = "bugs.gentoo.org" in url
```

We can be simpler, even if not 100% quality check